src/main/java/com/tetiana/tournament/repository/TeamRepository.java

```diff
@@ -0,0 +1,32 @@
+package com.tetiana.tournament.repository;
+
+import com.tetiana.tournament.domain.Team;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class TeamRepository {
+  public static List<Team> TEAMS = new ArrayList<>();
```

Comment:
Upper case variable should be static and final. Otherwise you should use lower case name.